assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -87,17 +91,19 @@ const AdUnits = ( {
 
 	const credentialsInputFile = useRef( null );
 
+	const [ isRemoving, setIsRemoving ] = useState( false );
+
 	return (
 		<>
 			{ serviceData.status.can_connect ? (
 				<>
 					{ isDisplayingNetworkMismatchNotice && (
 						<Notice
 							noticeText={ __(
-								'Your GAM network code is different than the network code the site was configured with. Editing has been disabled.',
+								'Your GAM network code is different than the network code the site was configured with.',
```

Comment:
Would love to see a bit more clarity on what this means, e.g.:

```suggestion
								'The GAM network code from your uploaded credentials differ from the previously saved network code. Legacy ad units associated with the prior network code will no longer work.',
```

Reply:
> Legacy ad units associated with the prior network code will no longer work.

This is not _exactly_ what happens. Because of [this change](https://github.com/Automattic/newspack-ads/pull/184/files#diff-727e5225ecc01afcf95934ec600179f9c67bb561963530a15966d643a18f99dbL328-L331), the ad units will continue to load on the frontend, but the requested network code will be the one coming from the credentials.

The likely result of this is the ad unit not working because it wouldn't find the appropriate combination of size and code to load from on the new network.

Reply:
I suggest we change it to:

> Your GAM network code is different than the network code the site was configured with. Legacy ad units are likely to not load.

Reply:
That sounds good to me—communicates an important nuance.